### PR TITLE
@damassi => Add some missing analytics

### PR DESCRIPTION
**Pull request by mzikherman**
Testing these with local Force/Reaction linked.

Feel free to merge or I can keep adding some commits for more analytics. I'm going from the top down in the spreadsheet.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d9f9dc21aefd208c7ae29f1f1b57af553d19eb77

https://deploy-preview-1806--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx [hunk 1]
```diff
@@ -148,6 +150,17 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
...
       })
+
+    const { tracking } = this.props
+    const trackingData = {
+      context_module: "Sidebar",
+      subject: "Contact Gallery",
+      type: "Button",
+      artwork_id: this.props.artwork._id,
+      action: "Click",
+    }
```
Comment (damassi) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx]: @mzikherman - do you plan on doing a pass using our new format? All of these string values need to go in `Artsy/Analytics/Schema` (some values are already there) and the tracking should to be attached via a decorator.

Comment (mzikherman) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx]: I thought this was the new way? (vs mediator triggered in Force).

I can move the string values to the schema file (I'm realizing as I'm going along they belong there), but the decorator stuff is really confusing to me. Is there something objectionable to this more imperative version? It seems a lot clearer.

Comment (damassi) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx]: I agree that this seems a little simpler in practice, but we've been moving tracking to decorators over the past year, most other code (except publishing) follows that format, and the same model is in place in Emission. I already fought this battle and lost if you feel like digging through the issues.

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarArtists.tsx [hunk 2]
```diff
@@ -34,6 +35,13 @@ export class ArtworkSidebarArtists extends React.Component<ArtistsProps> {
         <FollowArtistButton
           artist={artist}
           user={user}
+          trackingData={{
+            modelName: Schema.OwnerType.Artist,
+            context_module: Schema.ContextModule.Sidebar,
+            context_page: "Artwork page",
+            entity_id: artist._id,
+            entity_slug: artist.id,
+          }}
```
Comment (damassi) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarArtists.tsx]: In this instance this is how we do tracking with Follow, so no need for decorator here 👍

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx [hunk 3]
```diff
@@ -143,7 +143,14 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
     this.setState({ isErrorModalOpen: false })
   }
 
-  handleInquiry = () => {
+  @track((props: ArtworkSidebarCommercialContainerProps) => ({
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx]: Used the props param here (but still needed to type it myself).

**Comment by mzikherman** [comment]
I think this is ready to go, addresses several of the issues and uses the new analytics style.

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx (hunk 3 above)
Comment (damassi) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx]:
```js
@track<ArtworkSidebarCommercialContainerProps>(props => { ... })
```

Comment (damassi) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx]: (if you need state, the second arg to the generic takes state

Comment (mzikherman) [reply src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx]: Oh nice, will update to the templated version.

**Pull request merged**